x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/types.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum CardSelectorListItemAssetType {
+  video = 'video',
+  image = 'image',
+}
+
+export interface CardAssetType {
+  asset: {
+    type: CardSelectorListItemAssetType;
+    source: string;
+    alt: string;
+  };
+}
```

Comment:
Actually, what about this:

```suggestion
export enum CardAssetType {
  video = 'video',
  image = 'image',
}

export interface CardAsset {
  type: CardAssetType;
  source: string;
  alt: string;
}

export interface CardSelectorAssetListItem extends CardSelectorListItem {
  asset CardAsset;
}
```

And
- Use `CardSelectorAssetListItem` for the cards configurations.
- Use  `CardSelectorListItem` for the `CardSelectorList` component
- Use  `CardAsset` for the `OnboardingCardContentAssetPanel` component

Reply:
Thanks, this approach it's cleaner 💯